radis/lbl/loader.py

```diff
@@ -810,9 +823,9 @@ def fetch_databank(
             dbformat = "exomol-radisdb"  # downloaded in RADIS local databases ~/.radisdb  # Note @EP : still WIP.
         if exomol_database != None:
             assert source == "exomol"
-        if parfuncfmt == "exomol" and source != "exomol":
+        if [parfuncfmt, source].count("exomol") == 1:
             raise NotImplementedError(
-                "ExoMol partition functions must be used with ExoMol database"
+                "ExoMol partition functions must be used with ExoMol database - and vice-versa"
```

Comment:
Actually it worked the other way around, but you're right, let's be consistent and enforce consistency!